src/zope/app/container/browser/contents.py

```diff
@@ -204,6 +205,11 @@ def renameObjects(self):
         ids = request.get("rename_ids")
         newids = request.get("new_value")
 
+        if isinstance(ids, six.string_types):
+            ids = [ids]
+        if isinstance(newids, six.string_types):
+            newids = [newids]
```

Comment:
Ahh, sticking the magic `:list` on the end of the form names solved this problem.